README.md

```diff
@@ -27,18 +27,53 @@ Another moving part is the new-ish Annotations 2.0 which uses Leaflet.Illustrate
 
 ## Quick install
 
-We provide an install script for Amazon's Cloud9 service, which provides standard Linux virtual machines with a web-based editor: https://aws.amazon.com/cloud9/
+We provide an install script for Amazon's Cloud9 service, which provides standard Linux virtual machines with a web-based editor: https://c9.io/login
+
 
 To use it:
 
+### Installation (do this only once):
+
 1. Click `Create new workspace`
+
 2. Name your workspace and enter `https://github.com/USERNAME/mapknitter` (where `USERNAME` is your GitHub username) under `Clone from Git or Mercurial URL`
 3. Choose `Ruby` template
+
 4. Click `Create workspace`
-5. Once it loads, in the `bash` console at the bottom of the screen, type `source install_cloud.sh`
-6. Press `Control-x` when prompted for a message.
-7. Wait for the installation to run, and when it's complete, press the `Run` button at the top of the page.
-8. Open the URL which pops up to see MapKnitter booted up. Great work!
+
+5. Once it loads in the `bash` console at the bottom of the screen, pull in the latest `main` branch (note: AWS only lets us track `master` by default. Since we're deprecating in a month, quick workaround):
+
+```Bash
+$ git fetch --all
+
+$ git checkout remotes/origin/main
+
+$ git checkout -b main
+
+$ git remote add upstream https://github.com/publiclab/mapknitter.git 
+
+$ git pull upstream main
```

Comment:
Yes works. Pls try again.

Reply:
Nope doesn't work for me :man_shrugging: 
I tried cloning a fresh copy. I'm still having conflicts after merging

Reply:
Ok wait so 

$ git fetch --all
$ git checkout remotes/origin/main     - **this puts you on your remote origin**
$ git checkout -b main                         - **this creates a copy of it** 
$ git remote add upstream https://github.com/publiclab/mapknitter.git   
$ git pull upstream main                       - **if your remote origin main is up to date with remote upstream main, then there should be no merge conflict right?** 

So is it because your origin main is behind upstream main? Or am I missing something else

Reply:
if thats the case, then this wouldn't happen if you did the cloud9 setup right after forking. and these steps are for first time setup. so thats ok right?

Reply:
Hi @kaustubh-nair, your fork's main has additional commits - https://github.com/kaustubh-nair/mapknitter/tree/main, maybe that's why you are getting merge conflicts? 

@sashadev-sky how about force pulling so others don't get conflicts? Thanks!

Reply:
Oohh sorry my bad. I didn't notice that